packages/create-toolpad-app/src/templates/nextjs-pages/indexPage.ts

```diff
@@ -0,0 +1,25 @@
+const indexPage = `import Link from 'next/link';
+import * as React from 'react';
+import Typography from '@mui/material/Typography';
+import Box from '@mui/material/Box';
+
+export default function HomePage() {
+  return (
+    <Box
+      sx={{
```

Comment:
TBH, if we feel the need to add any styles here, it feels like we're doing something wrong with `PageContainer`. How does it look when this is just a `<div>`?

Reply:
This is to be able to center the message on the screen; we can remove the styles from the `div` and instead move it to the `<Typography>`

Reply:
I would just not center the message